src/widgets/freespacebar.cpp

```diff
@@ -194,7 +195,11 @@ void FreeSpaceBar::DrawText(QPainter *p, const QRect r) {
   if (additional_ > 0) {
     labels << Label(TextForSize(additional_text_, additional_), kColorAdd1);
   }
-  labels << Label(TextForSize(free_text_, free_ - additional_), kColorBg2);
+  if (free_ > additional_ || additional_ == 0) {
+    labels << Label(TextForSize(free_text_, free_ - additional_), kColorBg2);
+  } else {
```

Comment:
```suggestion
  }
  else {
```

Please follow same code style as the rest, please correct and squash commit.